client.go

```diff
@@ -57,15 +56,15 @@ type (
 
 var (
 	// Version is a API SDK version, sent in User-Agent header.
-	Version = "1.0"
+	Version = "1.1"
 
 	// DefaultBaseURL specifies base URL which will be used for calls unless
 	// other is specified in the Client struct.
 	DefaultBaseURL = "https://api.smartling.com"
 
 	// DefaultHTTPClient specifies default HTTP client which will be used
 	// for calls unless other is specified in the Client struct.
-	DefaultHTTPClient = http.Client{Timeout: 120 * time.Second}
+	DefaultHTTPClient = http.Client{}
```

Comment:
Will you able to configure this client from the App? From my experience it is better to always set explicit timeouts.

Reply:
We don't need time out here at all.